tests/Earthfile

```
@@ -565,6 +566,15 @@ fail-invalid-artifact-test:
     DO +RUN_EARTHLY --earthfile=fail-invalid-artifact.earth --should_fail=true --target="--artifact +test/foo /tmp/stuff" \
         --output_contains="cannot save artifact +test/foo, since it does not exist"
 
+wildcard-build:
+    COPY --dir wildcard-build .
+    DO +RUN_EARTHLY --earthfile=wildcard-build.earth --target=+wildcard-build
+    RUN cat earthly.output | acbgrep 'hello world 1'
+    RUN cat earthly.output | acbgrep 'hello world 2'
+    RUN cat earthly.output | acbgrep 'hello world 3'
+    RUN cat earthly.output | (! acbgrep 'not-test run')
```

```suggestion
    RUN cat earthly.output | acbgrep -v 'not-test run'
```

this will cause it to print an error stating the command failed, e.g.

```
alex@surfperch:~/gh/earthly/earthly$ echo "not-test run" | ./scripts/acbtest/acbgrep -v 'not-test run'
grep '-v' 'not-test run' failed
alex@surfperch:~/gh/earthly/earthly$ echo $?
1
```

where as using `(! ...` defeats the purpose of acbgrep (which is to print out a helpful message when it fails), 

for example, the output of a pass is confusing:

```
alex@surfperch:~/gh/earthly/earthly$ echo "blah" | (! ./scripts/acbtest/acbgrep 'not-test run')
blah
grep 'not-test run' failed
alex@surfperch:~/gh/earthly/earthly$ echo $?
0
```